app/Http/Controllers/Front/CheckoutController.php

```diff
@@ -102,14 +106,13 @@ public function checkoutForm(Request $request)
         }
         if (\Session::has('items')) {
             $content = \Session::get('items');
-            $attributes = $this->getAttributes($content);
+            $taxConditions = $this->getAttributes($content);
         } else {
             $content = Cart::getContent();
-            $attributes = $this->getAttributes($content);
+            $taxConditions = $this->getAttributes($content);
```

Comment:
else is not required, remove duplicate statements

Reply:
Fixed